clvm/operators.py

```diff
@@ -23,11 +23,26 @@
 )
 
 KEYWORDS = (
-    ". q . a i c f r l x = sha256 + - * divmod "
-    "substr strlen point_add pubkey_for_exp concat . > >s "
-    "logand logior logxor lognot ash lsh "
-    "not any all "
-    "softfork div "
+    # core opcodes 0x01-x08
+    ". q a i c f r l x "
+
+    # opcodes on atoms as strings 0x09-0x0f
+    "= >s sha256 substr strlen concat . "
+
+    # opcodes on atoms as ints 0x10-0x17
+    "+ - * / divmod > ash lsh "
```

Comment:
Might want to leave a blank spot in "opcodes on ints" namespace, just for aesthetics, if we add one later.

Hmm, come to think of it, will we specify that new opcodes are in a certain range?